pennylane/templates/embeddings/basis.py

```diff
@@ -15,12 +15,11 @@
 Contains the ``BasisEmbedding`` template.
```

Comment:
Sorry, I snuck in this overdue change, which makes ``BasisEmbedding`` independent from ``BasisState``.